packages/xo-web/src/xo-app/hub/recipes/recipe.js

```diff
@@ -0,0 +1,120 @@
+import * as ComplexMatcher from 'complex-matcher'
+import _ from 'intl'
+import ActionButton from 'action-button'
+import ButtonLink from 'button-link'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import marked from 'marked'
+import React from 'react'
+import { Card, CardBlock, CardHeader } from 'card'
+import { escapeRegExp } from 'lodash'
+import { form } from 'modal'
+import { connectStore } from 'utils'
+import { createGetObjectsOfType } from 'selectors'
+import { createKubernetesCluster } from 'xo'
+import { injectState, provideState } from 'reaclette'
+import { success } from 'notification'
+import { withRouter } from 'react-router'
+
+import RecipeForm from './recipe-form'
+
+const RECIPE_INFOS = {
+  id: '05abc8a8-ebf4-41a6-b1ed-efcb2dbf893d',
+  name: 'Kubernetes cluster',
+  description:
+    'Creates a Kubernetes cluster composed of 1 master and a configurable number of nodes working for the master.',
+}
```

Comment:
Is it temporary? Will it come from the server in the future?

Reply:
indeed, it's temporary